### PR TITLE
Add babers way

### PR DESCRIPTION
**Pull request by cdolan**

### CODE REVIEW

File: scripts/memes.coffee [hunk 1]
```diff
@@ -15,6 +16,9 @@ module.exports = (robot) ->
     console.log msg
     msg.send 'http://i.imgur.com/vQja8jB.jpg'
 
+  robot.hear /\bbaber\'?s\s?way\b/i, (msg) ->
```
Comment (aripollak) [inline scripts/memes.coffee]: I think the backslash before single quote is unneeded?

**Comment by aripollak** [comment]
I don't even realize this phrase was a thing until I searched history for it.

File: scripts/memes.coffee (hunk 1 above)
Comment (cdolan) [inline scripts/memes.coffee]: I usually escape anything that isn't alphanumeric out of habit. Not needed.

**Comment by aripollak** [comment]
babersway

**Comment by cdolan** [comment]
Merged

**Pull request closed without merging**